database/wireless_regdb.ksy

```diff
@@ -0,0 +1,189 @@
+meta:
+  id: wireless_regdb
+  title: Linux wireless regulation database (aka Central Regulatory (Domain Agent) DataBase)
+  license: copyleft-next-0.3.0 AND ISC
+  endian: be
+  encoding: ASCII
+  xref:
+    wikidata: Q11375
```

Comment:
`wikidata` is not supposed to point to arbitrary Wikipedia topics loosely associated with this data format (e.g. "wireless network" in this case). It is supposed to point exactly to the the Wikidata entry associated with this format. Otherwise, it's better *not* to specify it.

Reply:
NB also: 23aca49

Reply:
I have set this because https://en.wikipedia.org/wiki/Central_Regulatory_Domain_Agent redirects there. Can remove.